configuration/device-tree.md

```diff
@@ -448,9 +448,60 @@ When assigning to the `reg` property, the address portion of the parent node nam
 
 The `name` property is a pseudo-property - it shouldn't appear in a DT, but assigning to it causes the name of its parent node to be changed to the assigned value. Like the `reg` property, this can be used to give nodes unique names.
 
-
 <a name="part2.2.10"></a>
-#### 2.2.10 Examples
+#### 2.2.10 The overlay map file
+
+The introduction of the Pi 4, built around the BCM2711 SoC, brought with it many changes; some of these changes are additional interfaces, and some are modifications to (or removals of) existing interfaces. There are new overlays intended specifically for the Pi 4 that don't make sense on older hardware, e.g. overlays that enable the new SPI, I2C and UART interfaces, but other overlays don't apply correctly even though they control features that are still relevant on the new device.
+
+There is therefore a need for a method of tailoring an overlay to multiple platforms with differing hardware. Supporting them all in a single .dtbo file would require heavy use of hidden ("dormant") fragments and a switch to an on-demand symbol resolution mechanism so that a missing symbol that isn't needed doesn't cause a failure. A simpler solution is to add a facility to map an overlay name to one of several implementation files depending on the current platform.
+
+The overlay map, which is rolling out with the switch to Linux 5.4, is a file that gets loaded by the firmware at start of day. It is written in DTS source format - `overlay_map.dts`, compiled to `overlay_map.dtb` and stored in the overlays directory.
```

Comment:
Maybe non-technical people would get confused by the "start of day" phrase? Perhaps "by the firmware at bootup." instead?
(but then again, I suppose non-technical people aren't going to be reading this part of the documentation anyway? :rofl: )

Reply:
And there was me trying to humanise the language a little.

Reply:
Device tree is not for humans.

Reply:
OK - I'll tweak a few things, stripping out any remaining soul, and add a patch to squash in at merge time.

Reply:
Done.